script.js

```diff
@@ -1,4 +1,124 @@
+// User pick paper, Computer picks scissors, Computer WIN
+// User pick Stone, Computer picks scissors, User WIN
+// User pick Stone, Computer picks paper, Computer WIN
+
+// User pick scissors VS Computer picks paper, scissors, rock
+// var userPickS = function (input) {
+//   if (input == "scissors" && comOptions == 1) {
+//     var myOutputValue = "User WINS!";
+//   } else if (input == "scissors" && comOptions == 2) {
+//     var myOutputValue = "Computer WINS!";
+//   } else if (input == "scissors" && comOptions == 0) {
+//     var myOutputValue = "DRAW!";
+//   }
+//   return myOutputValue;
+// };
+
+// User pick paper VS Computer picks paper, scissors, rock
+// var userPickP = function (input) {
+//   if (input == "paper" && comOptions == 1) {
+//     var myOutputValue = "DRAW!";
+//   } else if (input == "paper" && comOptions == 2) {
+//     var myOutputValue = "User WINS!";
+//   } else if (input == "paper" && comOptions == 0) {
+//     var myOutputValue = "Computer WINS!";
+//   }
+//   return myOutputValue;
+// };
+
+// User pick rock VS Computer picks paper, scissors, rock
+// var userPickR = function (input) {
+//   if (input == "rock" && comOptions == 1) {
+//     var myOutputValue = "Computer WINS!";
+//   } else if (input == "rock" && comOptions == 2) {
+//     var myOutputValue = "DRAW";
+//   } else if (input == "rock" && comOptions == 0) {
+//     var myOutputValue = "User WINS!";
+//   }
+//   return myOutputValue;
+// };
+
+// Function for AI to have a random Math to select the word
+var aiOptions = function () {
+  var comOptions = Math.floor(Math.random() * 3);
+  console.log(comOptions);
+  if (comOptions == 0) {
+    var myOutputValue = "scissors";
+  } else if (comOptions == 1) {
+    var myOutputValue = "paper";
+  } else if (comOptions == 2) {
+    var myOutputValue = "rock";
+  }
+  return myOutputValue; // Return a value
+};
+
+// User pick VS Computer Pick
+var userPick = function (input) {
+  var comOptions = aiOptions();
+  var userOption = input.toLowerCase();
+  console.log(input, comOptions);
+  if (userOption == "rock" && comOptions == "paper") {
+    var myOutputValue = "Computer WINS!";
+  } else if (userOption == "rock" && comOptions == "scissors") {
+    var myOutputValue = "User WINS!";
+  } else if (userOption == "scissors" && comOptions == "paper") {
+    var myOutputValue = "User WINS!";
+  } else if (userOption == "scissors" && comOptions == "rock") {
+    var myOutputValue = "Computer WINS!";
+  } else if (userOption == "paper" && comOptions == "rock") {
+    var myOutputValue = "User WINS!";
+  } else if (userOption == "paper" && comOptions == "scissors") {
+    var myOutputValue = "Computer WINS!";
+  } else if (userOption == comOptions) {
+    var myOutputValue = "DRAW!";
+  } else {
+    return (myOutputValue = "Please type again, only scissors, paper or rock.");
+  }
+  return myOutputValue;
+};
+
+// Showing them together
+
 var main = function (input) {
```

Comment:
great work keeping your main function succinct by putting your game logic in userPick function